tests/test_manager_edgestore.py

```diff
@@ -0,0 +1,624 @@
+import os
+import tempfile
+from collections import Counter
+
+import pybel
+from pybel.constants import *
+from pybel.manager import models, Manager
+from pybel.utils import hash_dump
+from tests import constants
+from tests.constants import (
+    BelReconstitutionMixin,
+    test_bel_simple,
+    test_bel_simple_b,
+    expected_test_simple_metadata,
+)
+from tests.mocks import mock_bel_resources
+
+
+class TestEdgeStore(BelReconstitutionMixin):
+    """Tests that the cache can be queried"""
+
+    def setUp(self):
+        self.test_connection = os.environ.get('PYBEL_TEST_CONNECTION')
+
+        if self.test_connection:
+            self.connection = self.test_connection
+        else:
+            self.fd, self.path = tempfile.mkstemp()
+            self.connection = 'sqlite:///' + self.path
+
+        log.info('Test generated connection string %s', self.connection)
+
+        self.manager = Manager(connection=self.connection)
+        self.manager.create_all()
```

Comment:
Use the mixins from tests.constants instead of rewriting all of this code, then use an override of `setUp` with a call to `super(TestEdgeStore, self).setUp()` so you can also have `self.setup_graph_and_network()` at the end